### ISSUE TITLE
Can not download play-json4s-test-jackson_2.11;0.4.0

### ISSUE THREAD

**Issue by shunjikonishi** [tototoshi/play-json4s#11]
Now I'm working on updating playframework to 2.4.2.

I could update play-json4s with following setting.

```
  "com.github.tototoshi" %% "play-json4s-jackson" % "0.4.0",
```

But I couldn't update test module.
If I add following line, it cause dependency error.

```
  "com.github.tototoshi" %% "play-json4s-test-jackson" % "0.4.0",
```

I found this error in sbt output. Maybe there is something wrong with uploaded file on maven repo.

```
[warn] problem while downloading module descriptor: https://repo1.maven.org/maven2/com/github/tototoshi/play-json4s-test-jackson_2.11/0.4.0/play-json4s-test-jackson_2.11-0.4.0.pom: invalid sha1: expected=805f5efa912d43d641494f639bfc2f83f9bbf0bf computed=e567f53daa68ea72545e3401eac0983510e1860c (684ms)
```

**Comment by shunjikonishi** [comment]
It could solve with following.

build.sbt
``` scala
resolvers ++= Seq(
  Resolver.sonatypeRepo("public")
)
```

**Comment by natemurthy** [comment]
I've come across a similar problem with the checksums not matching. This command also sufficed as a workaround:

```
sbt 'set checksums := Nil' update 'set checksums := Seq("sha1", "md5")'
```

**Comment by timohirt** [comment]
@tototoshi this is still not working. Please update the sha.

**Comment by tototoshi** [comment]
This may be a sonatype's problem. 
I newly published 0.4.1, please try it.
